src/Util/ApiLoader.php

```diff
@@ -71,22 +71,14 @@ public function doRequest($endpoint, callable $callback)
             throw new RuntimeException($failureMessage . 'Error code ' . $response->getStatusCode());
         }
 
-        if (!in_array('application/json', $response->getHeader('Content-Type') ?? [])) {
-            throw new RuntimeException($failureMessage . 'Response is not JSON');
-        }
-
-        $responseBody = json_decode($response->getBody()->getContents(), true);
+        $responseJson = json_decode($response->getBody()->getContents(), true);
 
-        if (empty($responseBody)) {
+        if (empty($responseJson)) {
             throw new RuntimeException($failureMessage . 'Response could not be parsed');
         }
 
-        if (!isset($responseBody['success']) || !$responseBody['success']) {
-            throw new RuntimeException($failureMessage . 'Response returned unsuccessfully');
-        }
```

Comment:
This might still be relevant for the `ModuleHealthLoader` which also calls this method. Validate if that endpoint has a `'success'` key in its result and if it does, make sure this is still handled for that endpoint and the test is restored for it.

Reply:
Updated